### PR TITLE
Fix/frame and style overhaul

### PR DESCRIPTION
**Pull request by linuslundahl**
* Cleanup of `:help` and `:play` (guides), using components instead of raw html.
* Move `:help` and `:play` into `./documentation/*` instead.
* Update `<Carousel />` style.
  * Add [slide / nbr of slides] and show number on indicator-hover.
  * Make style coherent with the frame style.
* Add `Fira Code` as the new monospace-font.
* Add dynamically rendered `:help commands` section.

`:help commands`:
Kapture 2019-06-26 at 15 29 48

Second iteration carousel nav:
image

First iteration carousel nav:
image
image

Fira Code in the command console:
image

### CODE REVIEW

File: src/browser/documentation/guides/start.jsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+            </svg>
+          </div>
+          <ul className='topic-bullets'>
+            <li>What is a graph database?</li>
+            <li>How can I query a graph?</li>
+            <li>What do people do with Neo4j?</li>
+          </ul>
+          <div className='clearfix' />
+        </div>
+        <button play-topic='concepts' className='btn btn-cta'>
```
Comment (oskarhane) [inline src/browser/documentation/guides/start.jsx]: These buttons do not work, nothing happens when I click on them

Comment (oskarhane) [reply src/browser/documentation/guides/start.jsx]: Maybe tests can be added to lock this behavior?

**Comment by ryguyrg** [comment]
I like the overall design improvements, however there remains a lot of work to do here to make sure our current sandbox browser guides (consumed by ~7,300 unique users per month) display properly and have a good UX. 

I've sent a bunch of feedback to Oskar on Slack based on his ask for feedback, but neglected to read the line where he asked for feedback to be posted here!  It's not clear where changes need to be made (in the code or the guides), so happy to work with you folks if we can have some guidelines/templates.

image
image
image

**Comment by jexp** [comment]
I also find the UX for the nav is inconsistent with the rest of browser, as we  nowhere else have those rounded dark gray UI elements.

neo4j@bolt___ws-10-0-1-17-35485 neo4jsandbox com_443 - Neo4j Browser-2

**Comment by jexp** [comment]
In the build I got from Oskar, the buttons of the built-in start-frames don't work anymore.

https://dist-wkyvxdkmbw.now.sh/

neo4j@bolt___ws-10-0-1-17-35485 neo4jsandbox com_443 - Neo4j Browser-3

**Comment by jexp** [comment]
Not able to take a screenshot of it, but for guides who have a vertical scrollbar, that scrollbar overlaps with the nav-buttons (OSX, Chrome).

Also is the nav meant to scroll away?
Did you consider putting the nav in the bottom padding of the frame so it could be stable without affecting the content?

**Comment by oskarhane** [comment]
I think the slide navigation have disappeared for remote guides in the latest version. 
Both `:play reco` and `:play stackoverflow` are supposed to have multiple slides.

I like the new `:help commands` 🚀 
But, as you mentioned, we should review what's in there before shipping that :)

**Comment by oskarhane** [comment]
New built version: https://dist-lnrqnbuthe.now.sh

**Comment by oskarhane** [comment]
Latest build @jexp @linuslundahl: https://dist-pjzpkwigac.now.sh

**Comment by jexp** [comment]
Thanks a lot!

**Comment by jexp** [comment]
Looks good, I tried the guides of some sandboxes and so far they are good.

One thing for vertical scroll:
- it would be good if further down content would be indicated, sometimes it's not obvious
- when going to the next slide it would be nice to have the scroll reset to 0 so the slide is on top

neo4j@bolt___ws-10-0-1-82-34514 neo4jsandbox com_443 - Neo4j Browser

On the final slide of the PP the play icon is somehow off, but I'm not sure if that's a regression or ...?

`:play https://guides.neo4j.com/sandbox/icij-panama-papers/index.html`

neo4j@bolt___ws-10-0-1-82-34514 neo4jsandbox com_443 - Neo4j Browser-1

**Comment by linuslundahl** [comment]
@jexp 

Regarding the buttons in the panama papers, that's bad formatting of the buttons...

the html is:

```
<a play-topic="https://guides.neo4j.com/sandbox/icij-panama-papers/examples.html" style="display: flex;" class="clicked">
  <i class="fa fa-play-circle-o" style="padding-right:4px"></i>
  <button click-to-code="':play https://offshoreleaks-data.icij.org/offshoreleaks/neo4j/guide/examples.html'" class="btn btn-cta" style="background: #B3490E;padding:1em;color:#fff;">Run Queries</button>
</a>
```

Which means it's a button within a link... the correct formatting should be:

```
<button click-to-code="':play https://offshoreleaks-data.icij.org/offshoreleaks/neo4j/guide/examples.html'" class="btn btn-cta" style="background: #B3490E;padding:1em;color:#fff;">
  <i class="fa fa-play-circle-o" style="padding-right:4px"></i> Run Queries
</button>
```

**Comment by oskarhane** [comment]
Please rebase this one

**Pull request merged**